core/src/main/java/org/jboss/jandex/Indexer.java

```diff
@@ -1904,6 +1951,14 @@ private boolean processConstantPool(DataInputStream stream) throws IOException {
                         annoAttributes[pos] = HAS_MODULE_MAIN_CLASS;
                     } else if (len == RECORD_LEN && match(buf, offset, RECORD)) {
                         annoAttributes[pos] = HAS_RECORD;
+                    } else if (len == RUNTIME_INVISIBLE_ANNOTATIONS_LEN && match(buf, offset, RUNTIME_INVISIBLE_ANNOTATIONS)) {
+                        annoAttributes[pos] = HAS_RUNTIME_INVISIBLE_ANNOTATION;
+                    } else if (len == RUNTIME_INVISIBLE_PARAM_ANNOTATIONS_LEN
+                            && match(buf, offset, RUNTIME_INVISIBLE_PARAM_ANNOTATIONS)) {
+                        annoAttributes[pos] = HAS_RUNTIME_INVISIBLE_PARAM_ANNOTATION;
+                    } else if (len == RUNTIME_INVISIBLE_TYPE_ANNOTATIONS_LEN
+                            && match(buf, offset, RUNTIME_INVISIBLE_TYPE_ANNOTATIONS)) {
+                        annoAttributes[pos] = HAS_RECORD;
```

Comment:
Whoops :-)

Reply:
I wish I could blame this on the rebase... fixed now.